Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM golang:1.15.13 AS build
-ARG version=v2
+FROM golang:1.17.7 AS build
+
+# Set mpi-operator version
+# Defaults to v2
+ARG VERSION=v2
 
 ADD . /go/src/github.com/kubeflow/mpi-operator
 WORKDIR /go/src/github.com/kubeflow/mpi-operator
-RUN make mpi-operator.v1 mpi-operator.v2
```

Comment:
Why? 
this patch proposes you can choose which one you want by setting the right env vars 
I don't see why keeping them in a single image, is there a use case for it?

Reply:
You can certainly change the way we distribute, but you might risk breaking some users when they upgrade versions. kubeflow manifests were using `:latest` at some point :upside_down_face:

Reply:
> So that we only have one image in the registry per release that can be used for both.

Oh I see, to pull only one 


>kubeflow manifests were using :latest at some point upside_down_face

Yeah tons of fun on this one.